app/Entities/Repos/PageRepo.php

```diff
@@ -148,6 +148,12 @@ public function getNewDraftPage(Entity $parent)
             $page->book_id = $parent->id;
         }
 
+        if ($page->book->defaultTemplate) {
+            $page->forceFill([
+                'html'  => $page->book->defaultTemplate->html,
+            ]);
+        }
+
```

Comment:
We do need to support markdown also here, and probably the original used editor.
I suspect it should just be a case of also copying the `markdown` and `editor` values from the original page model.